client/src/unifyfs-sysio.c

```diff
@@ -40,6 +40,10 @@
  * Please also read this file LICENSE.CRUISE
  */
 
+#include <sys/stat.h>
```

Comment:
sys/stat and config.h are already included by unifyfs_internal.h

Reply:
fixed